File: XBOOK/XBOOK.Web/ClientApp/src/app/modules/_shared/models/accountbalance/accountbalance-view.model.ts
import { BaseModel } from '@shared/model/base.model';
export class AccountBalanceView extends BaseModel {
  accNumber: string;
  accName: string;
  debitOpening: number;
  creditOpening: number;
  debit: number;
  credit: number;
  debitClosing: number;
  creditClosing: number;
}
export class AccountBalanceViewModel {
  accNumber: string;
  accName: string;
  debitOpening: number;
  creditOpening: number;
  debit: number;
  credit: number;
  debitClosing: number;
  creditClosing: number;
}

File: XBOOK/XBOOK.Dapper/ViewModels/AccountBalanceViewModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace XBOOK.Dapper.ViewModels
{
    public class AccountBalanceViewModel
    {
        public string accNumber { get; set; }
        public string accName { get; set; }        
        public decimal debitOpening { get; set; }
        public decimal creditOpening { get; set; }
        public decimal debit { get; set; }
        public decimal credit { get; set; }
        public decimal debitClosing { get; set; }
        public decimal creditClosing { get; set; }

    }
   
}
